src/persistentSubscription/utils/persistentSubscriptionSettings.ts

```diff
@@ -81,24 +74,54 @@ export interface PersistentSubscriptionSettings {
    * The strategy to use for distributing events to client consumers.
    * @default ROUND_ROBIN
    */
-  strategy: ConsumerStrategy;
+  consumerStrategyName: ConsumerStrategy;
 }
 
-export const persistentSubscriptionSettingsFromDefaults = (
-  changes: Partial<PersistentSubscriptionSettings> = {}
-): PersistentSubscriptionSettings => ({
+export interface PersistentSubscriptionSettings
+  extends PersistentSubscriptionSettingsGeneric {
+  /**
+   * The exclusive position in the stream or transaction file the subscription should start from.
+   * @default END
+   */
+  startFrom: typeof START | typeof END | bigint;
+}
+
+export interface PersistentSubscriptionToAllSettings
+  extends PersistentSubscriptionSettingsGeneric {
+  /**
+   * The exclusive position in the stream or transaction file the subscription should start from.
+   * @default END
+   */
+  startFrom: typeof START | typeof END | Position;
+}
+
+const defaults: PersistentSubscriptionSettingsGeneric = {
   resolveLinkTos: false,
-  extraStats: false,
-  fromRevision: START,
+  extraStatistics: false,
   messageTimeout: 30_000,
   maxRetryCount: 10,
   checkpointAfter: 2_000,
```

Comment:
nitpick, not sure how strict we want to be 😬 
```suggestion
  checkPointAfter: 2_000,
```